app/js/views/settings.js

```diff
@@ -26,72 +23,63 @@ export default class SettingsView extends View {
     super(controller);
 
     this.modal  = this.$('gaia-modal');
-    this.addons = this.$('gaia-list');
+    this.list = this.$('gaia-list');
+    this.mergeButton = this.$('button[data-action="toggleMerging"]');
 
-    this.on('click', 'gaia-button', this._handleClick.bind(this));
-    this.on('click', 'button', this._handleClick.bind(this));
+    this.on('click', 'button', e => this._handleButtonClick(e));
+    this.on('click', 'gaia-button', e => this._handleButtonClick(e));
 
-    // this.on('change', 'gaia-switch', this._handleChange.bind(this));
+    this.list.addEventListener('click', e => this._handleListClick(e));
   }
 
   template() {
     return settingsViewTemplate;
   }
 
-  setAddons(addons) {
-    this.addons.innerHTML = '';
+  setMerging(merging) {
+    if (merging) {
```

Comment:
I kinda left this unfinished since I plan on handling the "Merge" functionality in another bug.